src/components/Loader.jsx

```diff
@@ -1,11 +1,11 @@
 import React, { Component } from 'react';
-import '../styles/Loader.css';
+import '../styles/Loader.scss';
 
 export default class Loader extends Component {
   render() {
     return (
       <div className="container loader-wrap"> 
-        <div className="loader">Loading...</div>
+      <div className="lds-ring"><div></div><div></div><div></div><div></div></div>
```

Comment:
can't stop getting surprised by that :) probably try some css approach, but i am fine with that as well